environment-dev.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
   - openbabel>=3.0.0
   - foyer>=0.11.3
   - forcefield-utilities>=0.2.1
-  - gsd>=2.0
+  - gsd>2.0,<3.0
```

Comment:
Why are we pinning gsd<3? I feel like we should want the latest version to have compatibility with hoomd 4.